tools/autograd/gen_variable_type.py

```diff
@@ -138,10 +161,21 @@
 # base at::Type
 FALLTHROUGH_RETURN_TYPES = {'int64_t', 'void*', 'bool', 'IntList'}
 FALLTHROUGH_FUNCTIONS = {
-    'eye', 'linspace', 'logspace', 'tensor', 'ones', 'ones_like', 'rand',
-    'randn' 'randperm', 'range', 'tensor', 'zeros', 'zeros_like',
+    'arange', 'eye', 'linspace', 'logspace', 'tensor', 'ones', 'ones_like',
+    'rand', 'randn', 'randperm', 'range', 'tensor', 'uniform', 'zeros',
+    'zeros_like', 'set_',
+    # these are only implemented on integral types
```

Comment:
why only for them?

Reply:
Uh nevermind. I didn't realize these are only logical ops (I though basic magic methods are there too)